src/Directory.Build.props

```diff
@@ -1,8 +1,8 @@
 <Project>
   <PropertyGroup>
-    <Version>1.0.17</Version>
-    <RuntimeFrameworkVersion>3.1.4</RuntimeFrameworkVersion>
-    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <Version>1.0.18</Version>
```

Comment:
this version will only bump after the release of the nuget no?

Reply:
Reverting back to .17